go.mod

```diff
@@ -5,7 +5,7 @@ go 1.23.7
 require (
 	github.com/go-logr/logr v1.4.2
 	github.com/google/safetext v0.0.0-20230106111101-7156a760e523
-	github.com/gophercloud/gophercloud/v2 v2.6.0
+	github.com/gophercloud/gophercloud/v2 v2.0.1-0.20250311123527-1031a8731f40
```

Comment:
This is suspicious, are you switching to the v3 (master) branch? We need to stay on v2 for the time being. If your patch is not in a v2 release, ask the gophercloud team.

Reply:
I don't think its the v3 master branch, I did `go get gophercloud/v2@commit` so it should be based on v2 ? Still, its a good idea to include it as part of a release, I will reach out to the gophercloud team.

Reply:
We should probably change, they added to v2 4 days ago see  https://github.com/gophercloud/gophercloud/pull/3335

Reply:
Done, thanks Iury!